koku/api/settings/test/tags/mappings/test_view.py

```diff
@@ -338,3 +339,8 @@ def test_order_by_fake_value(self):
         url = url + "?order_by[parent]=FAKE"
         response = self.client.get(url, **self.headers)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_parse_list(self):
+        """Test that we can parse a list."""
+        self.assertEqual(["A", "B"], parse_list("['A', 'B']"))
+        self.assertEqual(["A"], parse_list("A"))
```

Comment:
Add some scenarios for unexpected data to make sure it does the right thing such as `'some[long],str[ing]'` or passing in unexpected data types.

Reply:
@samdoran In our current implementation I don't think those kind combination would be possible, since the value is being built into the list by the serializer. Do you mind if I update testing in the multi filter PR instead so that I can get this one in? I have a feeling some of your suggestions may make this function obsolete, but would like to remove the ast sooner than latter.

Reply:
Yup, that's fine. I was wondering if the serializer would prevent a ceratain amount of bad data from getting here.